config/routes.rb

```diff
@@ -4,7 +4,7 @@
              path_names: {
                sign_in: 'api/v1/users/sessions',
                sign_out: 'logout',
-               registration: 'signup'
+               registration: 'api/v1/users'
```

Comment:
to avoid adding `api/v1/` to every route, you can use the method `api_version`, which specifies routes for the app version. Read about it in the docs, but it's ok if you don't use it here.